### PR TITLE
Add video tutorial link to background task docs

### PR DESCRIPTION
**Pull request by brunoAltinet**

### CODE REVIEW

File: src/docs/reference/modules/BackgroundTasks/README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # Background Tasks (`OrchardCore.BackgroundTasks`)
 
-This module provides tools to manage background tasks. This includes an admin UI to show which background tasks are registered with the ability to enable and disable them. 
\ No newline at end of file
+This module provides tools to manage background tasks. This includes an admin UI to show which background tasks are registered with the ability to enable and disable them. 
+
```
Comment (hishamco) [inline src/docs/reference/modules/BackgroundTasks/README.md:4]: Please check how we add video section in other modules

**Review by hishamco — CHANGES_REQUESTED** [review]

File: src/docs/reference/modules/BackgroundTasks/README.md (hunk 1 above)
Comment (Skrypt) [reply src/docs/reference/modules/BackgroundTasks/README.md:4]: I'm not sure about what you mean @hishamco 
I just looked at how we add Youtube videos in our doc and it is pretty much like he did there.

Comment (hishamco) [reply src/docs/reference/modules/BackgroundTasks/README.md:4]: https://github.com/OrchardCMS/OrchardCore/blob/532222e1a4d492281ca9dd1990f237d90e8ed382/src/docs/reference/modules/Setup/README.md#video

Comment (Skrypt) [reply src/docs/reference/modules/BackgroundTasks/README.md:4]: https://docs.orchardcore.net/en/main/docs/reference/modules/Setup/

**Comment by Piedone** [comment]
Thank you, but this will be done with the recommended formatting in https://github.com/OrchardCMS/OrchardCore/pull/15475/files#diff-b7bb2de822d93dccbdd2e4446fa466f6ec7d5eebc9edb86dc95474f06d1915f2.

**Pull request closed without merging**